src/sections/Projects/Sistent/components/divider/code.js

```diff
@@ -0,0 +1,306 @@
+    </Divider>
+    {content}
+</Root>`,
+];
+
+export const DividerCode = () => {
+  const location = useLocation();
+  const { isDark } = useStyledDarkMode();
+
+  return (
+    <SistentLayout title="Divider">
+      <div className="content">
+        <a id="Identity">
+          <h2>Divider</h2>
+        </a>
+        <div className="filterBtns">
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/divider"
+                ? "active"
+                : ""
+            }
+            onClick={() => navigate("/projects/sistent/components/divider")}
+            title="Overview"
+          />
+          <TabButton
+            className={
+              location.pathname ===
+              "/projects/sistent/components/divider/guidance"
+                ? "active"
+                : ""
+            }
+            onClick={() =>
+              navigate("/projects/sistent/components/divider/guidance")
+            }
+            title="Guidance"
+          />
+          <TabButton
+            className={
+              location.pathname === "/projects/sistent/components/divider/code"
+                ? "active"
+                : ""
+            }
+            onClick={() =>
+              navigate("/projects/sistent/components/divider/code")
+            }
+            title="Code"
+          />
+        </div>
+        <div className="main-content">
+          <p>
+            The Divider component adds a subtle line to separate content
+            sections, supporting horizontal/vertical orientation and
+            customizable alignment for text or icons. Ideal for enhancing layout
+            clarity.
+          </p>
+          <a id="Variant">
+            <h2>Variant Example</h2>
+          </a>
+          <p>
+            The Divider component supports three variants: fullWidth (default),
+            inset, and middle.
+          </p>
+          <div className="showcase">
+            <div className="items">
+              <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+                <List sx={style}>
+                  <ListItem>
+                    <ListItemText primary="Full width variant below" />
+                  </ListItem>
+                  <Divider component="li" />
+                  <ListItem>
+                    <ListItemText primary="Inset variant below" />
+                  </ListItem>
+                  <Divider variant="inset" component="li" />
+                  <ListItem>
+                    <ListItemText primary="Middle variant below" />
+                  </ListItem>
+                  <Divider variant="middle" component="li" />
+                  <ListItem>
+                    <ListItemText primary="List item" />
+                  </ListItem>
+                </List>
+              </SistentThemeProvider>
+            </div>
+            <CodeBlock name="variant-example" code={codes[0]} />
```

Comment:
Curious if it would make this more readable if instead of using array position for codes we use some map w/ a meaningful id so that if someone has to change it they will get some context of what each element in the array is.